cmd/example-tests/types.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/onsi/ginkgo/v2/types"
+)
+
+// copied directly from github.com/openshift/origin/test/extended/util/cluster/cluster.go
+type ClusterConfiguration struct {
```

Comment:
I took this over from k8s-tests, which duplicates it from origin.  It's the primary data we use to determine if we skip/run particular tests. Currently unused here.  Either way 3 copies isn't great.

Do test implementers just use our annotations in the test name `[Skipped:Proxy]`? If so we don't need this and let origin decide.  But then it means new criteria would have to be added to origin.

If we let them decide what to return in `list` by examining the cluster, then we'd maybe want to move this here along with the code to populate it, but we'll be pulling in a fair amount of kube dependencies.